config/core/test-configs.yaml

```diff
@@ -691,12 +694,14 @@ device_types:
     filters:
       - passlist: {defconfig: ['x86-chromebook']}
 
-  hp-x360-12b-n4000-octopus:
+  hp-x360-12b-n4000-octopus: &n4000-octopus
```

Comment:
Please move this to a different PR.  Also I think we need to review the octopus device type names as we may not need to keep the 2 variants.